Description: Fix backslash characters in docstrings
 Use @xbackslashchar to produce backslashes in docstrings consistently and be
 compatible with Texinfo 6.
Origin: upstream, http://hg.savannah.gnu.org/hgweb/octave/rev/2ec049e50ed8
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=790533
Last-Update: 2015-07-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff -r 3300cb76cccd -r 2ec049e50ed8 libinterp/corefcn/utils.cc
--- a/libinterp/corefcn/utils.cc	Tue Jul 07 19:09:35 2015 -0700
+++ b/libinterp/corefcn/utils.cc	Thu Jul 09 21:44:09 2015 -0400
@@ -1379,7 +1379,7 @@
 \n\
 Implementation Note: Strings are first converted to double values before the\n\
 checks for valid indices are made.  Unless a string contains the NULL\n\
-character @nospell{\"\\0\"}, it will always be a valid index.\n\
+character @nospell{\"@xbackslashchar{}0\"}, it will always be a valid index.\n\
 @end deftypefn")
 {
   octave_value retval;
diff -r 3300cb76cccd -r 2ec049e50ed8 scripts/io/beep.m
--- a/scripts/io/beep.m	Tue Jul 07 19:09:35 2015 -0700
+++ b/scripts/io/beep.m	Thu Jul 09 21:44:09 2015 -0400
@@ -20,9 +20,9 @@
 ## @deftypefn {Function File} {} beep ()
 ## Produce a beep from the speaker (or visual bell).
 ##
-## This function sends the alarm character @qcode{"\a"} to the terminal.
-## Depending on the user's configuration this may produce an audible beep,
-## a visual bell, or nothing at all.
+## This function sends the alarm character @qcode{"@xbackslashchar{}a"} to
+## the terminal. Depending on the user's configuration this may produce an
+## audible beep, a visual bell, or nothing at all.
 ## @seealso{puts, fputs, printf, fprintf}
 ## @end deftypefn
 
